examples/spring-streams-example/build.gradle.kts

```diff
@@ -0,0 +1,92 @@
+import com.google.protobuf.gradle.*
+
+plugins {
+  alias(libs.plugins.spring.plugin)
+  idea
+  application
+  id("com.google.protobuf") version "0.8.19"
+  id("io.spring.dependency-management") version "1.1.6"
+  id("org.springframework.boot") version "3.3.4"
```

Comment:
should be already in the libs 🤔

Reply:
gradle/libs.versions.toml

spring-boot = { id = "org.springframework.boot", version.ref = "spring-boot" }
spring-boot = "2.7.18"

Reply:
I've solved it in the commit.